Build error on fresh docker, undefined references in `rpc`

**Describe the bug**
On Linux, building with a fresh docker image

**To Reproduce**
Steps to reproduce the behavior:
1. Git clone
2. docker compose + docker make
3. Errors in log

**Expected behavior**
A successfully built image in `dist`

**Logs**
```
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/applications/rpc/rpc_app.o: in function `rpc_system_app_lock_status_process':
/project/applications/rpc/rpc_app.c:66: undefined reference to `pb_release'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/applications/rpc/rpc_gui.o: in function `rpc_system_gui_stop_screen_stream_process':
/project/applications/rpc/rpc_gui.c:125: undefined reference to `pb_release'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/applications/rpc/rpc_gui.o: in function `rpc_system_gui_free':
/project/applications/rpc/rpc_gui.c:375: undefined reference to `pb_release'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/applications/rpc/rpc.o: in function `rpc_send':
/project/applications/rpc/rpc.c:681: undefined reference to `pb_encode_ex'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /project/applications/rpc/rpc.c:685: undefined reference to `pb_ostream_from_buffer'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /project/applications/rpc/rpc.c:687: undefined reference to `pb_encode_ex'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/applications/rpc/rpc.o: in function `rpc_send_and_release':
/project/applications/rpc/rpc.c:704: undefined reference to `pb_release'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/applications/rpc/rpc.o: in function `rpc_send_and_release_empty':
/project/applications/rpc/rpc.c:715: undefined reference to `pb_release'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/applications/rpc/rpc.o: in function `rpc_session_worker':
/project/applications/rpc/rpc.c:559: undefined reference to `pb_release'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /project/applications/rpc/rpc.c:499: undefined reference to `pb_decode_ex'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `archive_extract_foreach_cb':
/project/lib/toolbox/tar/tar_archive.c:224: undefined reference to `mtar_eof_data'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /project/lib/toolbox/tar/tar_archive.c:225: undefined reference to `mtar_read_data'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_open':
/project/lib/toolbox/tar/tar_archive.c:87: undefined reference to `mtar_init'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_free':
/project/lib/toolbox/tar/tar_archive.c:94: undefined reference to `mtar_is_open'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /project/lib/toolbox/tar/tar_archive.c:95: undefined reference to `mtar_close'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_get_entries_count':
/project/lib/toolbox/tar/tar_archive.c:116: undefined reference to `mtar_foreach'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_dir_add_element':
/project/lib/toolbox/tar/tar_archive.c:124: undefined reference to `mtar_write_dir_header'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_finalize':
/project/lib/toolbox/tar/tar_archive.c:129: undefined reference to `mtar_finalize'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_file_add_header':
/project/lib/toolbox/tar/tar_archive.c:148: undefined reference to `mtar_write_file_header'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_file_add_data_block':
/project/lib/toolbox/tar/tar_archive.c:157: undefined reference to `mtar_write_data'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_file_finalize':
/project/lib/toolbox/tar/tar_archive.c:162: undefined reference to `mtar_end_data'
/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: .obj/f7-firmware//project/lib/toolbox/tar/tar_archive.o: in function `tar_archive_unpack_to':
/project/lib/toolbox/tar/tar_archive.c:248: undefined reference to `mtar_foreach'
```

This was on branch `dev`

--- a/ReadMe.md
+++ b/ReadMe.md
@@ -7,6 +7,13 @@
 Welcome to [Flipper Zero](https://flipperzero.one/)'s Firmware repo!
 Our goal is to create nice and clean code with good documentation, to make it a pleasure for everyone to work with.
 
+# Clone the Repository
+
+You should clone with 
+```shell
+$ git clone --recursive https://github.com/flipperdevices/flipperzero-firmware.git
+```
+
 # Update firmware
 
 [Get Latest Firmware from Update Server](https://update.flipperzero.one/)
@@ -73,6 +80,8 @@ docker-compose exec dev make
 Check `dist/` for build outputs.
 
 Use **`flipper-z-{target}-full-{suffix}.dfu`** to flash your device.
+
+If compilation fails, make sure all submodules are all initialized. Either clone with `--recursive` or use `git submodule update --init --recursive`.
 
 # Build on Linux/macOS
 

